### PR TITLE
Mozilla Hubs case

### PR DESCRIPTION
**Pull request by josevalim**
Screenshot_2021-06-02 Social virtual spaces with Elixir at Mozilla

### CODE REVIEW

File: _posts/2021-06-02-social-virtual-spaces-with-elixir-at-mozilla.markdown [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+layout: post
+title: Social virtual spaces with Elixir at Mozilla
+author: José Valim
+category: Elixir in Production
+excerpt: A case study of how Elixir is being used at Mozilla.
+logo: /images/cases/logos/mozilla-hubs.png
+tags: virtual-spaces phoenix
+---
+
+*Welcome to our series of case studies about companies using Elixir in production. [See all cases](/cases.html) we have published so far.*
```
Comment (fertapric) [inline _posts/2021-06-02-social-virtual-spaces-with-elixir-at-mozilla.markdown:11]: I believe this is the correct form, but not 100% sure 😅 

```suggestion
*Welcome to our series of case studies about companies using Elixir in production. [See all the cases](/cases.html) we have published so far.*
```

Comment (josevalim) [reply _posts/2021-06-02-social-virtual-spaces-with-elixir-at-mozilla.markdown:11]: I think we don't need the article as we are not specifying any cases in particular? 🤔

File: _posts/2021-06-02-social-virtual-spaces-with-elixir-at-mozilla.markdown [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+[Hubs](https://hubs.mozilla.com/) is Mozilla's take on virtual social experiences. You build your own private spaces and share them with your friends, co-workers, and community. Avatars in this space can move freely in a 3D social environment and watch videos, exchange messages, and talk to other people nearby. All you need is a browser and a microphone!
+
+Hubs is [fully open source](https://github.com/mozilla/hubs). You can also host it on your infrastructure via [Hubs Cloud](https://hubs.mozilla.com/cloud). Community managers, educators, and event organizers have been using Hubs Cloud to run virtual events and online activities tailored to their specific brandings and needs. All it takes to run your own version of Hubs is a button click - which perhaps makes Hubs the most deployed Phoenix application ever!
```
Comment (fertapric) [inline _posts/2021-06-02-social-virtual-spaces-with-elixir-at-mozilla.markdown:15]:
```suggestion
Hubs is [fully open source](https://github.com/mozilla/hubs). You can also host it on your infrastructure via [Hubs Cloud](https://hubs.mozilla.com/cloud). Community managers, educators, and event organizers have been using Hubs Cloud to run virtual events and online activities tailored to their specific brandings and needs. All it takes to run your own version of Hubs is one click away - which perhaps makes Hubs the most deployed Phoenix application ever!
```

File: _posts/2021-06-02-social-virtual-spaces-with-elixir-at-mozilla.markdown [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+![Mozilla Hubs](/images/cases/bg/mozilla-hubs.jpg)
+
+## From VR to Elixir
+
+The Hubs team started at Mozilla as the Mixed Reality team about 3.5 years ago. Their main goal was to explore ways for online social interaction via avatars and mixed reality.
+
+They quickly focused on building their first proof of concept, where avatars could communicate, move around, and join different rooms, everything running directly from the browser. This was a significant departure from the state of the art of Virtual Reality everywhere, as the getting started experience up to this point was cumbersome and often required steep investment in the form of VR headsets.
```
Comment (fertapric) [inline _posts/2021-06-02-social-virtual-spaces-with-elixir-at-mozilla.markdown:23]:
```suggestion
They quickly focused on building their first proof of concept, where avatars could communicate, move around, and join different rooms, everything running directly in the browser. This was a significant departure from the state of the art of Virtual Reality everywhere, as the getting started experience up to this point was cumbersome and often required steep investment in the form of VR headsets.
```

**Review by fertapric — APPROVED** [review]

File: _posts/2021-06-02-social-virtual-spaces-with-elixir-at-mozilla.markdown [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+excerpt: A case study of how Elixir is being used at Mozilla.
+logo: /images/cases/logos/mozilla-hubs.png
+tags: virtual-spaces phoenix
+---
+
+*Welcome to our series of case studies about companies using Elixir in production. [See all cases](/cases.html) we have published so far.*
+
+[Hubs](https://hubs.mozilla.com/) is Mozilla's take on virtual social experiences. You build your own private spaces and share them with your friends, co-workers, and community. Avatars in this space can move freely in a 3D social environment and watch videos, exchange messages, and talk to other people nearby. All you need is a browser and a microphone!
+
+Hubs is [fully open source](https://github.com/mozilla/hubs) and you can host it on your infrastructure via [Hubs Cloud](https://hubs.mozilla.com/cloud). Community managers, educators, and event organizers have been using Hubs Cloud to run virtual events and online activities tailored to their specific brandings and needs. All it takes to run your own version of Hubs is one click away - which perhaps makes Hubs the most deployed Phoenix application ever!
```
Comment (ericmj) [inline _posts/2021-06-02-social-virtual-spaces-with-elixir-at-mozilla.markdown:15]: Link to the elixir repo? https://github.com/mozilla/reticulum

**Review by ericmj — APPROVED** [review]

File: _posts/2021-06-02-social-virtual-spaces-with-elixir-at-mozilla.markdown (hunk 4 above)
Comment (josevalim) [inline _posts/2021-06-02-social-virtual-spaces-with-elixir-at-mozilla.markdown:15]:
```suggestion
Hubs is [fully](https://github.com/mozilla/hubs) [open source](https://github.com/mozilla/reticulum) and you can host it on your infrastructure via [Hubs Cloud](https://hubs.mozilla.com/cloud). Community managers, educators, and event organizers have been using Hubs Cloud to run virtual events and online activities tailored to their specific brandings and needs. All it takes to run your own version of Hubs is one click away - which perhaps makes Hubs the most deployed Phoenix application ever!
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**